python/tank/context.py

```diff
@@ -793,26 +784,17 @@ def to_dict(self):
 
         :returns: A dictionary representing the context.
         """
-        return self._to_dict()
```

Comment:
This change was caused by the revert of my original pickle fix that was merged some time ago in the Python 3 branch.